covid-vue-app/src/pages/StatsPage.vue

```diff
@@ -1,16 +1,121 @@
 <template lang="html">
   <v-container class="py-12 height_class">
     <h1>This is the Stats page</h1>
+    <svg class="barchart" :width="width + marginLeft / 2" :height="height + marginTop">
```

Comment:
Make a new component called `BarChart` in [here](https://github.com/dabreadman/Node-Podman-OpenShift-CI-CD/tree/emer_visualisedata/covid-vue-app/src/components).

Check out the other components to see what you have to do.

Reply:
I've added this and it works with yarn serve but doesn't work with docker-compose up

Reply:
Do you have the logs?
If I remember correctly it's something about dependencies?